<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="5">

  <testsuite name="raceLogin"
    errors="0" failures="1" hostname="" id="" package="raceLogin" skipped="4"
    tests="5" time="19.93" timestamp="Fri, 15 Apr 2016 12:02:47 GMT">
  
    <testcase name="should fail at login with bad password" time="19.93" assertions="3">

    

      <failure message="Testing if element &lt;#loginFormEmail&gt; contains text: &quot;javier@sportmaniacs.com&quot;.">    at Object.module.exports.should fail at login with bad password (C:\Users\Emilio\nightwhatch\src\tests\raceLogin.js:16:11)
    at Module.call (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\module.js:63:34)
    at C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\testcase.js:97:29
    at _fulfilled (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:796:13)
    at C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:125:13)
    at nextTickCallbackWith0Args (node.js:420:9)</failure>

    
    </testcase>
  
  
    
    <testcase
      name="should login succesfully">
      <skipped />
    </testcase>
    
    <testcase
      name="should logOut">
      <skipped />
    </testcase>
    
    <testcase
      name="should say the user has an email">
      <skipped />
    </testcase>
    
    <testcase
      name="should create an account">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
